v1/topdown/eval.go

```diff
@@ -1185,7 +1185,7 @@ func (e *eval) biunifyRef(a, b *ast.Term, b1, b2 *bindings, iter unifyIterator)
 			e:         e,
 			ref:       ref,
 			pos:       1,
-			plugged:   ref.Copy(),
+			plugged:   ref.CopyNonGround(),
```

Comment:
If we only have this one user of the new method, I suppose we could also make it a helper here? It's not using any internals, so that should work.

Update: ignore this, I've overlooked the second use in v1/ast/compile.go.